src/components/AnimatedPlanet/hooks/useAnimatedPlane.ts

```diff
@@ -0,0 +1,21 @@
+import { easings, SpringRef, SpringValue, useSpring } from "react-spring";
+
+interface AnimatedPlaneTrailReturnType {
```

Comment:
There are 3 occurrences of `interface AnimatedPlaneTrailReturnType` in the `/hooks` folder.
Two of them needs to be renamed